utils/land-patterns.stanza

```diff
@@ -1174,28 +1256,13 @@ public pcb-landpattern two-pin-chip-landpattern (length:Toleranced,
                                                  width:Toleranced,
                                                  lead-length:Toleranced,
                                                  lead-width?:False|Toleranced,
-                                                 fabrication-tolerance:Double,
-                                                 placement-tolerance:Double
                                                  density-level:DensityLevel,
                                                  polarized?:True|False):
   make-two-pin-chip-landpattern(
     length, width, lead-length, lead-width?,
-    fabrication-tolerance, placement-tolerance, 
     density-level, polarized?, polarized?
```

Comment:
is polarized passed twice here?

Reply:
Yes, intentionally. One flag controls whether the component is marked "polarized" and the other marks whether to use "a" and "c" as the pad names (versus "p[1]" and "p[2]))